index.js

```diff
@@ -1,15 +1,13 @@
 // Set up auth for Google spreadsheet
-var GoogleSpreadsheet = require('google-spreadsheet');
-var creds = require('./google-sheets-token.json');
-const { promisify } = require('util');
+const GoogleSpreadsheet = require('google-spreadsheet')
+const creds = require('./google-sheets-token.json')
```

Comment:
In modern JS, prefer using `const` and `let` over `var`. The difference between `const` and `let` is that variables defined as `const`s cannot be changed, while `let`s can be. 

Using the exclusively increases code clarity, as I can read the code and discern which variables are going to change (which in general should be avoided as much as possible) vs. which ones are going to stay constant, making the code easier to follow.